### ISSUE TITLE
[MediaLibrary] MediaLibrary.getAssetsAsync ignores `order` value with `default` key in `sortBy` field on ios

### ISSUE THREAD

**Issue by center-777** [expo/expo#3889]
Despite the `true` or a `false` of `order` value
```
MediaLibrary.getAssetsAsync({
    sortBy: [[ MediaLibrary.SortBy.default, true ]]
});
```
or
```
MediaLibrary.getAssetsAsync({
    sortBy: [[ MediaLibrary.SortBy.default, false ]]
});
```
we receive the same result.

### Environment

Expo CLI 2.10.1 environment info:
    System:
      OS: macOS 10.14.4
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 11.9.0 - /usr/local/bin/node
      Yarn: 1.13.0 - /usr/local/bin/yarn
      npm: 6.7.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Xcode: 10.2/10E125 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^32.0.0 => 32.0.5 
      react: 16.5.0 => 16.5.0 
      react-native: https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz => 0.57.1 
      react-navigation: ^3.2.3 => 3.2.3 
    npmGlobalPackages:
      expo-cli: 2.10.1

Example: https://snack.expo.io/@center-777/medialibrary

**Comment by cruzach** [comment]
Hey @center-777,

Could you create a small example Snack that reproduces the behavior?

Thanks!
-Charlie

**Comment by center-777** [comment]
Added example.
It seems that sorting does not work at all.

**Comment by lukmccall** [comment]
Was fixed - #4420. Also, It's worth mentioning that if you are using `default` sort, `ascending` property doesn't matter.
